tests/acceptance/features/bootstrap/Provisioning.php

```diff
@@ -463,7 +473,13 @@ public function userHasBeenCreatedWithDefaultAttributes(
 	 * @throws Exception
 	 */
 	public function userHasBeenCreatedWithDefaultAttributesAndWithoutSkeletonFiles(string $user):void {
-		$this->userHasBeenCreatedWithDefaultAttributes($user);
+		$response = $this->userHasBeenCreatedWithDefaultAttributes($user);
+		$actualStatusCode = $response->getStatusCode();
+		Assert::assertEquals(
+			'200',
+			$actualStatusCode,
+			"HTTP status code was $actualStatusCode User creation was not successful"
+		);
```

Comment:
cant we use the existing function `theHttpStatusCodeShouldBe()` ?

Reply:
its because theHttpStatusCodeShouldBe() checks from the response set early. and we are avoiding that in this pr

Reply:
I think we should be able to set `response` parameter in `theHttpStatusCodeShouldBe` method? of if the `response` parameter is `null` then the function should check from the response set early. Please check if that's how the function works, if not we might need to refactor the function `theHttpStatusCodeShouldBe` itself (but should be done in some other PR)

Reply:
/**
	 * Check that the status code in the saved response is the expected status
	 * code, or one of the expected status codes.
	 *
	 * @param int|int[]|string|string[] $expectedStatusCode
	 * @param string|null $message
	 *
	 * @return void
	 */
-------> this is the given description of the function which tells it checks on the saved response

Reply:
> I think we should be able to set `response` parameter in `theHttpStatusCodeShouldBe` method? only if the `response` parameter is `null` then the function should check from the response set early. Please check if that's how the function works, if not we might need to refactor the function `theHttpStatusCodeShouldBe` itself (but should be done in some other PR)

implemented in this PR https://github.com/owncloud/ocis/pull/7169